modules/exploits/windows/smb/ms17_010_psexec.rb

```diff
@@ -13,6 +13,8 @@
 #    Security Options >
 #     Network Access: Sharing and security model for local accounts
 
+require 'mitre/attack/technique'
```

Comment:
If we set up zeitwerk and our naming scheme correctly, we shouldn't need these explicit requires here

```suggestion
```